modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -795,15 +804,54 @@ Ransomware Protection (Enterprise Edition onyl)
 Marketplace URL: https://marketplace.owncloud.com/apps/ransomware_protection[Ransomware Protection]
 
 Use these commands to help users recover from a Ransomware attack. 
-You can find more information about the application in the documentation <../../enterprise/ransomware-protection/index>.
+You can find more information about the application in the documentation 
+<../../enterprise/ransomware-protection/index>.
 
-[width="100%",cols="50%,82%",]
+....
+sudo -u www-data occ ransomguard:scan <timestamp> <user>
+....
+
+Arguments
```

Comment:
Please set these as level 3 headers. That way, they'll stand out more.
![set arguments as a header](https://user-images.githubusercontent.com/196801/46282559-1f7c8d00-c572-11e8-83d5-6e9c6d41583d.png)

Reply:
You know that this means we have to to L3 **all** options and arguments headers in **both** documents - just to confirm...

Reply:
Yep. Give me a sec to see just how many are present.

Reply:
OK, there's about 37 in total. I'll get this sorted out.

Reply:
After reviewing the file, I found that there are a number of existing examples where "Arguments" is set as a level-4 header, such as https://github.com/owncloud/docs/blob/the_next_occ_fix/modules/ROOT/pages/configuration/server/occ_command.adoc#arguments-10. So, for consistency's sake, it's worth continuing this. I'll create a commit that gets this done.